client/src/index.js

```diff
@@ -1,4 +1,5 @@
 import 'materialize-css/dist/css/materialize.min.css'; // Imports CSS. Webpack will find this and load it.
+import 'react-toastify/dist/ReactToastify.min.css';
 import React from 'react';
```

Comment:
Webpack will find this CSS in the node_modules automatically and thus, import all of the styles that come with the toasts.